test/bindings.test.ts

```diff
@@ -405,9 +405,12 @@ describe('MongoCryptConstructor', () => {
 });
 
 describe('MongoCryptContext', () => {
-  const context = new MongoCrypt({
-    kmsProviders: serialize({ aws: {} })
-  }).makeDecryptionContext(serialize({}));
+  let context: MongoCryptContext;
```

Comment:
unrelated to the deps PR: the `test` action has been segfaulting consistently (not just for this PR, see the past runs: https://github.com/mongodb-js/mongodb-client-encryption/actions/workflows/test.yml).  


I reproduced locally just running this single test in a loop.  Instantiating a new context per-test resolves the issues locally.  it's not a permanent fix, I think we should look into this problem separately and maybe it's a libmongocrypt bug, but I wanted green CI for this and future PRs.